cluster-autoscaler/processors/provreq/injector.go

```diff
@@ -29,26 +29,34 @@ import (
 	provreqconditions "k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/conditions"
 	provreqpods "k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/pods"
 	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/provreqclient"
+	"k8s.io/autoscaler/cluster-autoscaler/provisioningrequest/provreqwrapper"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog/v2"
 	"k8s.io/utils/clock"
 )
 
 const (
-	defaultRetryTime = 10 * time.Minute
+	defaultRetryTime = 1 * time.Minute
```

Comment:
Please make it a flag.

Reply:
This is initial retry time. What the reasoning to make it configurable?

Reply:
There is context there: https://github.com/kubernetes-sigs/kueue/discussions/2931
Kueue got some requests for this retry time.

Reply:
@yaroslava-serdiuk I believe it's about https://github.com/kubernetes-sigs/kueue/discussions/2931#discussioncomment-10489551. But my previous comments stands, let's make it configurable and easy to mitigate. It's not like we have any data supporting this choice of hardcoded values AFAIK.

Reply:
The "right" initial value depends on (at least) 2 factors:

* size of the cluster and, so, the number of incoming provisioning requests. 
* the performance/throughput of provrequ processing.

We have no control over the first item, and the second item is being improved right now. So there is no easy way of knowing what value is good now, and, more importantly what value will be OK in 3 months. Having it hardcoded makes any adjustment/finetuning much harder.